src/status_im/constants.cljs

```diff
@@ -29,6 +29,10 @@
 (def ^:const contact-verification-status-trusted 5)
 (def ^:const contact-verification-status-untrustworthy 6)
 
+(def ^:const membership-status-pending 1)
```

Comment:
About these new constants:

- It seems we have have `status_im2/common/constants.cljs` that's preferred instead of `status_im/constants.cljs`. It looks like it's one of those files that were duplicated and now cause a lot of confusion...
- *Membership* is an overloaded word in our domain. Sometimes used for group chats, communities and for special invitations to group chats (just learned about this hidden concept recently). What do you think of giving them a more specific prefix, like `admin-membership-status-*`?

Reply:
- Moved the constants under `status_im2` 👍 
- Initially, I thought of the naming convention `admin-membership-status-*`, and the prefix would imply that these constants belong to AC. But, I believe this is more of a global constant like `content-type-*`, `contact-request-*`, and `contact-verification-status-*`.  I'm open to `community-membership-status-*` too. Let me know your thoughts.

Reply:
> I'm also open to community-membership-status-* as this belongs to communities. Let me know your thoughts.

If I'm not mistaken, all these type constants strictly follow the names and values defined in status-go. In status-go, there's a type called `ActivityCenterMembershipStatus` https://github.com/status-im/status-go/blob/04de916cba5775a6b2bb7a4b56460e1f9f136c34/protocol/activity_center.go#L32

Are the constants you defined in this PR based on that type in status-go? If so, then the most correct name would be something like `activity-center-membership-status-*`.

Please, ignore this suggestion if I got the wrong type from status-go.

Reply:
Apologies, I have not known about the naming convention that correlates with `status-go`. I have updated the names of constants to `activity-center-membership-status-*`. Thanks, @ilmotta 👍